Classes/DataProcessing/ContainerProcessor.php

```diff
@@ -132,6 +132,7 @@ protected function processColPos(
             /** @var ContentObjectRenderer $recordContentObjectRenderer */
             $recordContentObjectRenderer = GeneralUtility::makeInstance(ContentObjectRenderer::class);
             $recordContentObjectRenderer->start($child, 'tt_content');
+            $child['data'] = $child;
```

Comment:
This will create a copy of the array. So if original `$child` was:

```php
[
  'item1' => 1,
  'item2' => 2,
],
```

then after this line it will be:

```php
[
  'item1' => 1,
  'item2' => 2,
  'data' => [
      'item1' => 1,
      'item2' => 2,
    ],
  ],
],
```

This does not look right.

Reply:
i will think about it.

Reply:
i will close this PR, s. #146